agent/src/main/scala/za/co/absa/atum/agent/exception/AtumAgentException.scala

```diff
@@ -16,15 +16,35 @@
 
 package za.co.absa.atum.agent.exception
 
+/**
+ * This class represents an exception thrown by the Atum Agent.
+ *
+ * @param message A message describing the exception.
```

Comment:
I would even remove this line completely from all occurrences, not really bringing any value

Reply:
still todo potentially?

Reply:
It's a standard part of java/scala doc for a class with constructor parameters. I would rather keep it. Some exceptions might have different parameters (HttpException for instance).